### ISSUE TITLE
Add fontSize support to RatingBadge

### ISSUE THREAD

**Issue by ghost** [priceline/design-system#724]
RatingBadge currently supports fontWeight but not fontSize. Add fontSize support.

**Comment by sdalonzo** [comment]
Note: I created this with an account that no longer exists while evaluating GH Enterprise.

**Comment by rajat2502** [comment]
Hey, @sdalonzo @craigpalermo I would like to work on this issue if it is not taken.

**Comment by craigpalermo** [comment]
Hey @rajat2502, sure, thanks for volunteering! I think we were just waiting on a design from @wesleymartin85; any update there?

**Comment by rajat2502** [comment]
Thanks for replying @craigpalermo, Let me know when I should start working on the Issue.

**Comment by rajat2502** [comment]
Hey @craigpalermo, if this one is taking some time... are there any other issues or works for me to get comfortable with the project.

**Comment by craigpalermo** [comment]
Hey @rajat2502, we confirmed with @wesleymartin85 during our meeting on Friday that he's going to tackle the `needs design` stories this week, so you should have something to work with soon. Thanks for your patience! 😄

**Comment by rajat2502** [comment]
Thanks, till then I will work on some other issues.

**Comment by wesleymartin85** [comment]
@rajat2502 thank you for offering the support! I will get to this design ASAP. Monday, the 18th I should be able to get you some designs and style specs.

**Comment by craigpalermo** [comment]
@wesleymartin85 is this still something you'd like to have done?

**Comment by wesleymartin85** [comment]
@craigpalermo I think we can close this, the ratings badge has some flaws to it. It makes more sense to open a new ticket that covers those. I have all the design guidelines for what we would need to do to update this component so that its used in path. Now is the best time to do it since in Q4 hotels is refactoring the listings component. I am 99% sure we are not using this component because of its limitations and how the css is currently written.
